editor/modules/map/editor-state/index.ts

```diff
@@ -154,6 +153,7 @@ export class EditorState implements MouseHandler, KeyHandler {
         switch (this.state) {
             case State.Palette: next_state = this.palette_area.mouseMovePalette(event); break;
             case State.Editor: next_state = this.editor_area.mouseMoveEditor(event); break;
+            default: throw 'Unreachable code reached';
```

Comment:
Maybe adding some context and especially with this code is supposed to be unreachable could help further investigation